src/test/scala/io/druid/indexer/spark/TestScalaBatchIndexTask.scala

```diff
@@ -219,6 +220,7 @@ class TestScalaBatchIndexTask extends FlatSpec with Matchers
       *     taskPre should ===(task)
       */
     task.asInstanceOf[SparkBatchIndexTask].getDataSchema.getParser.getParseSpec should ===(taskPre.getDataSchema.getParser.getParseSpec)
+    task.asInstanceOf[SparkBatchIndexTask].getHadoopDependencyCoordinates.asScala.toSet should ===(Set("org.apache.spark:spark-core_2.10:1.6.1-mmx0"))
```

Comment:
ordering matters in the actual impl, but for the test using `Set` is the easiest check since it only uses one entry.